content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-1.0.md

```diff
@@ -74,13 +74,13 @@ The certificates API has changed:
 * To import a cluster-level certificate, use PUT with `cluster/<usage>` where `usage` is one of the following:
    * 'p2p-tls' — TLS certificate to be used in P2P communication
    * 'p2p-session' — session certificate to be used in P2P communication
-   * 'rpc-api-tls' — TLS certificate to be used in RPC API communication
+   * 'rest-api-tls' — TLS certificate to be used in REST API communication
```

Comment:
As far as I can see, this has yet to be updated:
https://github.com/corda/corda-runtime-os/blob/537c930e50deb0ebe13d15b2057c10a6459ddf31/components/membership/membership-rest/src/main/kotlin/net/corda/membership/rest/v1/CertificatesRestResource.kt#L43

@ben-millar is this part of your grand plan?